### PR TITLE
Include owner change information for .SE and .NU domains

### PR DESCRIPTION
**Pull request by juankuquintana**
This PR adds support pages for owner change for .SE and .NU domains

### CODE REVIEW

**Review by OleMchls — COMMENTED** [review]
The link is invalid Screenshot 2023-07-04 at 13 35 42

According to https://github.com/dnsimple/dnsimple-app/blob/main/app/views/contact_modifications/show.html.erb#L81 it should include the TLD.

Without any further research but sending an .SE or .NU trade form to **AU**_trades@ispapi.net seems suspicious.

---

As I've mentioned in our chat, exposing registrar specific instructions here, will cause issues as we are not exposing the domains registrar and we want to be able to support multi-tenancy.

**Comment by juankuquintana** [comment]
> The link is invalid Domain_PDF_Generator_-_Request
> 
> According to https://github.com/dnsimple/dnsimple-app/blob/main/app/views/contact_modifications/show.html.erb#L81 it should include the TLD.
> 
> Without any further research but sending an .SE or .NU trade form to **AU**[_trades@ispapi.net](mailto:_trades@ispapi.net) seems suspicious.
> 
> As I've mentioned in our chat, exposing registrar specific instructions here, will cause issues as we are not exposing the domains registrar and we want to be able to support multi-tenancy.

Do you want to setup a custom mail that is sent to us, we process it and re-send it to hexonet?

File: content/articles/domains-nu.markdown [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+This article explains the requirements and special procedures for .NU domain names.
+
+
+## Owner change of a .NU domain {#owner-change}
+
+When a .NU owner is changed (change registrant name, email, or organization) the owner must submit a signed request for it to proceed. Please follow the steps below.
+
+
+1. Download the corresponding [form](http://www.domainform.net/form/index.php).
```
Comment (juankuquintana) [inline content/articles/domains-nu.markdown:23]:
```suggestion
1. Download the corresponding [form](https://www.domainform.net/form/nu/search?view=ownerchange).
```

File: content/articles/domains-se.markdown [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+This article explains the requirements and special procedures for .SE domain names.
+
+
+## Owner change of a .SE domain {#owner-change}
+
+When a .SE owner is changed (change registrant name, email, or organization) the owner must submit a signed request for it to proceed. Please follow the steps below.
+
+
+1. Download the corresponding [form](http://www.domainform.net/form/index.php).
```
Comment (juankuquintana) [inline content/articles/domains-se.markdown:23]:
```suggestion
1. Download the corresponding [form](https://www.domainform.net/form/se/search?view=ownerchange).
```

**Pull request closed without merging**